marathon_quiz.cpp

```diff
@@ -415,6 +421,73 @@ void displayQuestionsByCategory(Question* head, int category){
 	printf("\n");
 }
 /*
+Display questions page
+*/
+void displayQuestionsInPages(Question* questionHead) {
```

Comment:
@Starfisheunyoung
I think there might be some issues with the logic of the function. After making the necessary changes to get the program to compile correctly, here are the problems I noticed:

- The functionality allows moving to the next page, but it doesn't work when trying to go back to the previous page; it just says "try again."
- The first question on the second page is numbered as '1' again, but it should continue from the previous page, showing 6, 7, 8, and so on.
- Lastly, the page numbers seem to be off. On the first page, it correctly displays "Page 1 of 2," but on the second page, it shows "Page 1 of 1."

You might want to check the `printPlayers` function, which also has a pagination system that works like this one should work!